File: programmers/lev1/68935.py
def solution(n):
    answer = 0
    n_gin=[]
    if n>=3:
        while n>=3:
            n_gin.append(n%3)
            n//=3
            if n<3:
                n_gin.append(n)
    else:
        n_gin.append(n)
    for i in range(len(n_gin),0,-1):
        answer+=n_gin[i-1]*(3**(len(n_gin)-i))
    
    return answer

n=45
n=2
n=0
n=5
print(solution(n))

File: java/src/programmers/year2023/lv1/_68935.java
package programmers.year2023.lv1;

public class _68935 {
    public static void main(String[] args) {
        int n = 125;
        solution(n);
    }

    public static int solution(int n) {
        int answer = 0;

        String[] num = Integer.toString(n, 3).split("");
        for (int i = 0; i < num.length; i++) {
            double tmp = Integer.parseInt(num[i]) * Math.pow(3, i);
            answer += tmp;
        }

        return answer;
    }
}
